src/app.js

```diff
@@ -520,15 +520,7 @@ var run = function () {
     } else {
       url = 'https://ethereum.github.io/solc-bin/bin/' + version;
     }
-    var isFirefox = typeof InstallTrigger !== 'undefined';
```

Comment:
I just want to check if running without workers fix all my tests in firefox. #277 seems to not fix the issue